### ISSUE TITLE
NVDA freezes while bulk deleting emails in Microsoft Outlook 2021

### ISSUE THREAD

**Issue by alexstine** [nvaccess/nvda#13883]
### Steps to reproduce:
1. Open Microsoft Outlook 2021.
2. Find some emails you do not want anymore. I deleted a bunch of old transactional emails from the Deleted Items folder.
3. Hold down Shift and press Down Arrow to select some messages. I selected 18 total.
4. Press the Delete key followed by Enter to confirm the permanent deletion.

### Actual behavior:
NVDA stops speaking and can sometime crash all together. When NVDA crashes at this stage, so does Outlook. As a result, Outlook goes through the roof with CPU usage. I know this is a bug with NVDA as the exact same steps work fine in JAWS 2021.

### Expected behavior:
I expect NVDA to not freeze up and crash Outlook.

#### NVDA installed/portable/running from source:
NVDA installed.

#### NVDA version:
2022.1.

#### Windows version:
Windows 10 Professional.

#### Name and version of other software in use when reproducing the issue:
Microsoft Outlook 2021.

#### Other information about your system:
See debug log attached.

#### Does the issue still occur after restarting your computer?
Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No.

#### If NVDA add-ons are disabled, is your problem still occurring?
No

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes.

[NVDA-Debug.log](https://github.com/nvaccess/nvda/files/9075918/NVDA-Debug.log)

**Comment by Brian1Gaff** [comment]
If you select Espeak does it still do it?
 Brian

-- 
***@***.***
Sent via blueyonder.(Virgin media)
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
----- Original Message ----- 
From: "Alex Stine" ***@***.***>
To: "nvaccess/nvda" ***@***.***>
Cc: "Subscribed" ***@***.***>
Sent: Saturday, July 09, 2022 6:22 AM
Subject: [nvaccess/nvda] NVDA freezes while bulk deleting emails in 
Microsoft Outlook 2021 (Issue #13883)

**Comment by seanbudd** [comment]
>#### If NVDA add-ons are disabled, is your problem still occurring?
>Yes.

I've changed this to no, and closing this issue as it is a problem with an add-on

**Comment by alexstine** [comment]
I just replicated this again this time with no add-ons enabled. Attaching a fresh log. NVDA froze for nearly 60 seconds when I was simply pressing Down Arrow in the Deleted Items list of Outlook 2021. This is certainly without a doubt some issue in NVDA.

Thanks.

[nvda.log](https://github.com/nvaccess/nvda/files/9128341/nvda.log)

**Comment by seanbudd** [comment]
@alexstine - trimmed logs are very difficult to debug, a full debug log is preferred. If this is a privacy issue, feel free to send it to info@nvaccess.org.
Your steps to reproduce also differ from the original issue - could you please update the issue?
Please also include the [specific version of Outlook](https://support.microsoft.com/en-us/office/what-version-of-outlook-do-i-have-b3a9568c-edb5-42b9-9825-d48d82b2257c) you are using.

**Comment by alexstine** [comment]
@seanbudd I found out a bit more while troubleshooting with Microsoft support. At a certain point, I could actually replicate this bug again with JAWS. I ended up trying a cleanviews command, repairing Outlook, and even fresh install. It seems like this only occurs on emails with a ton of graphics such as emails from Uber. After I removed those emails, I could scroll as normal. Super weird issue but I doubt it is related to NVDA.

My only suggestion would be maybe opening up a follow-up to not crash NVDA when other programs crash. Even though Outlook froze up and started to not respond with JAWS, I could still maintain control of my machine. This is not the case whenever programs crash with NVDA Outlook, Firefox, or anything else. If I can get a solid log in the future, I will open up a new issue.

For now, seems like the ball is in Microsoft's court and no one knows if they will actually care enough to fix it. We shall see. Just a few short years ago when everything was fine on Windows 7, stuff just worked. Now, it is like pulling teeth to get lesser results.

Thanks again.
